docs/source/llms/deployments/index.rst

```diff
@@ -0,0 +1,1100 @@
+.. _deployments:
+
+========================================
+MLflow Deployments Server (Experimental)
+========================================
+
+The MLflow AI Gateway service is a powerful tool designed to streamline the usage and management of
```

Comment:
are we going to replace these "AI Gateway" before merging similar to https://github.com/mlflow/mlflow/pull/10551 ?

Reply:
Yes, it's replaced now.